tests/pools/selection-strategies.test.js

```diff
@@ -67,7 +67,6 @@ describe('Selection strategies test suite', () => {
           max,
           './tests/worker-files/thread/testWorker.js',
           {
-            maxTasks: 1000,
             workerChoiceStrategy: 'UNKNOWN_STRATEGY'
```

Comment:
I think this could be merged into one line `{ workerChoiceStrategy: 'UNKNOWN_STRATEGY' }`

Reply:
the npm run lint:fix command put it in this way